src/backend/distributed/planner/multi_physical_planner.c

```diff
@@ -5226,6 +5226,45 @@ CompareShardPlacements(const void *leftElement, const void *rightElement)
 
 
 /*
+ * CompareShardPlacementsByNode compares two shard placements by their nodename
+ * and nodeport.
+ */
+int
+CompareShardPlacementsByNode(const void *leftElement, const void *rightElement)
+{
+	const ShardPlacement *leftPlacement = *((const ShardPlacement **) leftElement);
```

Comment:
Hmm, yeap that makes sense.

Originally, I wanted to keep it together with the other Compare function.